typedpreferences/src/main/java/info/metadude/android/typedpreferences/BooleanPreference.java

```diff
@@ -41,9 +23,7 @@ public BooleanPreference(final Preferences preferences, final String key) {
      * Boolean.
```

Comment:
Check the base lib's java doc method description and retain it..The previous review comment is not handled in all the places...Please check and handle in all the scenarios applicable